assets/js/vue/components/data-land/HelpModal.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <span>
+    <b-button class="sm rounded-circle" @click="openHelpModal" variant="primary">
+      <i class="fa fa-question-circle"></i>
```

Comment:
```suggestion
      <i class="fa fa-question-circle"></i>&nbsp; Help
```

Reply:
Instead of `&nbsp;` add `mr-1` to the class.